Source/KSCrash/Recording/KSCrash.m

```diff
@@ -470,7 +475,25 @@ - (void) doctorReport:(NSMutableDictionary*) report
     }
 }
 
-- (NSDictionary*) reportWithID:(int64_t) reportID
+- (NSArray*)reportIDs
+{
+    int reportCount = kscrash_getReportCount();
+    int64_t reportIDsC[reportCount];
+    reportCount = kscrash_getReportIDs(reportIDsC, reportCount);
+    NSMutableArray* reportIDs = [NSMutableArray arrayWithCapacity:(NSUInteger)reportCount];
+    for(int i = 0; i < reportCount; i++)
+    {
+        [reportIDs addObject:@(reportIDsC[i])];
+    }
+    return reportIDs;
+}
+
+- (NSDictionary*) reportWithID:(NSNumber*) reportID
```

Comment:
I'm not quite sure I follow why NSNumber is being used here... That would allow floats and booleans and other things that would be meaningless to this method.

Reply:
Because of the fact `reportIDs` method returns `NSArray *` that can hold `NSNumber`s but not `int64_t`s.

Reply:
Cool!